scipy/io/wavfile.py

```diff
@@ -384,6 +384,14 @@ def _read_fmt_chunk(fid, is_big_endian):
     # fmt should always be 16, 18 or 40, but handle it just in case
     _handle_pad_byte(fid, size)
 
+    if format_tag == WAVE_FORMAT.PCM:
+        if bytes_per_second != fs * block_align:
+            raise ValueError("WAV header is invalid: nAvgBytesPerSec must "
```

Comment:
Apparently now we ask the last space to be at the start of the line instead of being the last character.

Reply:
This is still in discussion there and would be a nice to have apparently #13955